src/project/internal/projectactionscontroller.cpp

```diff
@@ -1218,7 +1218,7 @@ bool ProjectActionsController::askIfUserAgreesToOpenCorruptedProject(const Strin
 
 void ProjectActionsController::warnProjectCannotBeOpened(const String& projectName, const std::string& errorText)
 {
-    std::string title = mtrc("project", "File “%1” is corrupted and cannot be opened").arg(projectName).toStdString();
+    std::string title = mtrc("project", "File "%1" is corrupted and cannot be opened").arg(projectName).toStdString();
```

Comment:
Needs to be
`std::string title = mtrc("project", "File `\"%1\" is corrupted and cannot be opened").arg(projectName).toStdString();`

Reply:
Done.